src/parsers/pluscal.ts

```diff
@@ -91,11 +94,49 @@ export class TranspilerStdoutParser extends ProcessOutputHandler<DCollection> {
         return true;
     }
 
+    
     private addError(location: vscode.Position, message: string) {
         const locRange = new vscode.Range(location, location);
         this.result.addMessage(this.filePath, locRange, message);
     }
 
+    /**
+     * Adds info on labels added by the pluscal translated.
+     * 
+     * Only takes effect if the `-reportLabels` was added to PlusCal options. Output looks like:
+     * 
+     *       The following labels were added: // 1
+     *           Lbl_1 at line 16, column 5
+     *           Lbl_2 at line 23, column 9
+     * 
+     * 
+     * So we can start looking for labels as soon as we see (1)
+     * and stop as soon as we stop seeing label strings.
+     */
+    private tryParseAddedLabels(line: string) {
+        if (line.startsWith("The following label")) { // could be `was added` or `were added`
```

Comment:
Why not use a regex here as well, considering you're already using a regex matcher further down?

Moreover, I'd suggest to add a link to the PCal method that emits these strings: https://github.com/tlaplus/tlaplus/blob/21f924cc0d77501ae2de104dda4f56997a29bc5b/tlatools/org.lamport.tlatools/src/pcal/ParseAlgorithm.java#L668-L682

Reply:
We need to use a regex to capture the name of the label below. Here we don't need to capture anything, so it makes sense to use something weaker. It also follows the existing convention on line 69 https://github.com/tlaplus/vscode-tlaplus/pull/350/files#diff-5395c8cd3da8e88aabad5e743d78366c8848a3a4ffa280ed77b08c02a0c6e7a4R69

Reply:
It is possible that Pcal will someday emit something like "The following label was removed".  In order to make the code more robust, how about: 

```typescript
const validPrefixes = ["The following labels were added:", "The following label was added:"];

if (validPrefixes.some(prefix => line.startsWith(prefix))) {
    ...
}
```

Reply:
Done!